pysat/_instrument.py

```diff
@@ -2703,7 +2748,7 @@ def _check_if_keywords_supported(func, **kwargs):
 
```

Comment:
Recommend returning void, and replacing "Returns" in docstring with "Raises" section, describing what is raised and why.